rootfs/Dockerfile

```diff
@@ -48,6 +48,8 @@ RUN apt-get update && apt-get install -y \
 	pv \
 	python \
 	python-dev \
+	libssl-dev \
```

Comment:
is there a particular reason these dependencies are now required? I guess azure and gcs client libs require it?